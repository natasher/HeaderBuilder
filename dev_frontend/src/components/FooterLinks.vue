<template>
  <footer class = "footer-links">
    <a href = "#">
      <font-icon icon = "info-circled" />
      Manual &amp; Support
    </a>

    <reset-builder v-once />
  </footer>
</template>

<script>
import FontIcon     from './BaseGeneric/FontIcon.vue'
import ResetBuilder from './ResetBuilder.vue'

export default {

  name: 'FooterLinks',

  components: {
    FontIcon,
    ResetBuilder,
  },

}
</script>
