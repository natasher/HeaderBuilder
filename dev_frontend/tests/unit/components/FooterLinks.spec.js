import { shallowMount } from '@vue/test-utils'
import FooterLinks from '../../../src/components/FooterLinks'

describe('FooterLinks.vue', () => {

  test('should match snapshot', () => {
    const wrapper = shallowMount(FooterLinks)

    expect( wrapper.element ).toMatchSnapshot()
  })

})