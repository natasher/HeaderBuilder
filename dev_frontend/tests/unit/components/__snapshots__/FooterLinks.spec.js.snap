// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterLinks.vue should match snapshot 1`] = `
<footer
  class="footer-links"
>
  <a
    href="#"
  >
    <fonticon-stub
      icon="info-circled"
    />
    
    Manual & Support
  
  </a>
   
  <resetbuilder-stub />
</footer>
`;
